pkg/controller/direct/gkehub/featuremembership_controller.go

```diff
@@ -249,7 +248,7 @@ func (a *gkeHubAdapter) Update(ctx context.Context, updateOp *directbase.UpdateO
 	log.V(2).Info("updating object", "u", u)
 	actual := a.actual.MembershipSpecs[a.membershipID]
 	//  There are no output fields in the api Object, so we can compare the desired and the actaul directly.
```

Comment:
Just curious, is this comment still valid?

Reply:
It still holds in some sense. but we need to rephrase it a bit, will send out a separate PR